config.py

```diff
@@ -16,7 +16,7 @@ class ConfigEmail:
 
 class Config:
     def __init__(self):
-        self.install_path = Path(os.getenv("TINYHIPPO_INSTALL_PATH"))
+        self.install_path = Path(os.getenv("TINYHIPPO_INSTALL_PATH", "/etc/tinyHIPPO"))
```

Comment:
I was getting an error for some reason and Martin suggested I add it

Reply:
If you were getting an error it was likely because the environment variable wasn't set in the testing environment you were on. So when you run the script you need to set the TINYHIPPO_INSTALL_PATH environment variable yourself or run the setup script beforehand. Please remove the default because we want the user to only have to change the value of this variable in one place in the code (the tinyhippo_setup script) and it would be overwritten here if this stays.